src/frontend/js/index.html

```diff
@@ -22,6 +22,30 @@
     float: right;
     font-size: small;
 }
+.gcvt-ctrl ul {
+    list-style-type: none;
+}
+.gcvt-ctrl ul li {
+    margin-bottom: 0;
+}
```

Comment:
If not using the Construct-UI widgets mentioned below, could these be `.flowlistholder ul` or inline styles instead? Want to restrict scope of change.